corehq/apps/reports/templates/reports/filters/location_async.html

```diff
@@ -4,9 +4,9 @@
      data-make-optional='{{ make_optional|JSON }}'
```

Comment:
Should this one also be changed?

Reply:
i addressed values that were causing issue in the ticket i.e locations list. make_optional is a bool value. Will address this in a follow up PR if needed.